html_xblock/html.py

```diff
@@ -28,6 +28,12 @@ class HTML5XBlock(StudioEditableXBlockMixin, XBlock):
         default=_('Text')
     )
     data = String(help=_('Html contents to display for this module'), default=u'', scope=Scope.content)
+    allow_javascript = Boolean(
+        display_name=_('Allow JavaScript execution'),
+        help=_('Whether js should be allowed or not in this module'),
+        default=False,
+        scope=Scope.settings
```

Comment:
I think it makes more sense to have `scope=Scope.content` here as this field would be needed for all users and across all runs of the course. See https://github.com/open-craft/XBlock/blob/master/xblock/fields.py#L156.

Reply:
You're right. Thanks @pkulkark :+1: